pelicun/assessment.py

```diff
@@ -1641,47 +1619,39 @@ def _loss__add_replacement_carbon(
     """
     rcarb = ('replacement', 'Carbon')
     if median is not None:
+        # TODO(JVM): in this case we need unit (add config parser check)
 
-        # TODO: in this case we need unit (add config parser check)
-
-        adf.loc[rcarb, ('Quantity', 'Unit')] = "1 EA"
+        adf.loc[rcarb, ('Quantity', 'Unit')] = '1 EA'
         adf.loc[rcarb, ('DV', 'Unit')] = unit
         adf.loc[rcarb, ('DS1', 'Theta_0')] = median
 
         if distribution is not None:
-
-            # TODO: in this case we need theta_1 (add config parser check)
+            # TODO(JVM): in this case we need theta_1 (add config parser check)
 
             adf.loc[rcarb, ('DS1', 'Family')] = distribution
             adf.loc[rcarb, ('DS1', 'Theta_1')] = theta_1
-    else:
-
-        # add a default replacement carbon value as a placeholder
-        # the default value depends on the consequence database
-
-        # for FEMA P-58, use 0 kg
-        if damage_process_approach == 'FEMA P-58':
-            adf.loc[rcarb, ('Quantity', 'Unit')] = '1 EA'
-            adf.loc[rcarb, ('DV', 'Unit')] = 'kg'
-            adf.loc[rcarb, ('DS1', 'Theta_0')] = 0
+    elif damage_process_approach == 'FEMA P-58':
```

Comment:
I removed some comens in cases where I thought that the code strongly speaks for itself. We could definitely put them back if needed.
At some point it would be nice to combine all add_XYZ functions into one, it would be more elegant and flexible.